src/ADSB/ADSBVehicleManager.cc

```diff
@@ -43,6 +43,10 @@ void ADSBVehicleManager::_cleanupStaleVehicles()
         if (adsbVehicle->expired()) {
             qCDebug(ADSBVehicleManagerLog) << "Expired " << QStringLiteral("%1").arg(adsbVehicle->icaoAddress(), 0, 16);
             _adsbVehicles.removeAt(i);
+
+            uint32_t key = adsbVehicle->icaoAddress();
```

Comment:
I don't get it.
The same code is in single line 50. :thinking:

Reply:
Omg. I swear when I looked I didn't see that. The fix was added 2 years ago: https://github.com/mavlink/qgroundcontrol/commit/30c227eb1a88a6df3ccd328b443d712515c84303

Sorry. This can be closed then :roll_eyes: